apps/marketing/src/app/(marketing)/open/typefully.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import type { HTMLAttributes } from 'react';
+
+import Link from 'next/link';
+
+import { FaXTwitter } from 'react-icons/fa6';
+
+import { cn } from '@documenso/ui/lib/utils';
+import { Button } from '@documenso/ui/primitives/button';
+
+export type TypefullyProps = HTMLAttributes<HTMLDivElement>;
+
+export const Typefully = ({ className, ...props }: TypefullyProps) => {
+  return (
+    <div className={cn('flex flex-col', className)} {...props}>
+      <h3 className="px-4 text-lg font-semibold">Twitter Stats</h3>
+
+      <div className="border-border mt-2.5 flex flex-1 items-center justify-center rounded-2xl border py-8 shadow-sm hover:shadow">
+        <div className="flex flex-col items-center gap-y-4 text-center">
+          <FaXTwitter className="h-12 w-12" />
```

Comment:
I assume that this will need some `text-foreground` css class to make sure it inverts as needed for light/dark mode? 🤔

Reply:
I dont think so, the logo didn't need any other CSS class apart from the sizing ones and the themes changed correctly 😶 
[Twitter-Logo.webm](https://github.com/documenso/documenso/assets/23498248/6376582b-0771-40ca-939e-155eb48906b8)